CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ Change Log
 **Improvements**
 
 - Changed `DJ08` check for `__str__` method to ignore abstract models (@denizdogan)
+- Added `DJ09` - Model must define `class Meta`
+- Added `DJ10` - Class Meta from Model has to define `verbose_name`
+- Added `DJ11` - Class Meta from Model has to define `verbose_name_plural`
+- Fixed some flake8 issues in test files (@avallbona)
+- Refactored some tests
```

Comment:
@avallbona you also have to add your name to all the other lines that have changes made by you

Reply:
done